### ISSUE TITLE
Upgrade assistant should support migrating to sdk-style project without changing target framework.

### ISSUE THREAD

**Issue by voroninp** [dotnet/upgrade-assistant#1501]
## Summary

There should be an option to convert a project from the old format to the sdk-style format without changing its target framework.

## Motivation and goals

I have a project which targets .NET 4.8 It is of the old format, and I'd like to convert it to the new sdk-style project. I do not want to switch to NetStandard or .NET 5+ in one go. But I want to have the benefits of manually editable proj file.

**Comment by grasbdo** [comment]
You can use the legacy version of the upgrade-assistant in version 0.4.421302, see [here](https://learn.microsoft.com/en-us/dotnet/core/porting/upgrade-assistant-install-legacy) for more details.

We also upgrade in two steps our solutions. In the first step, we upgrade to the new SDK style, then we try to remove all the unsupported libraries and upgrade to .net standard 2.0 or .net 6. 

Hope that helps.

**Comment by grasbdo** [comment]
Yes, that is true. We've updated our service references and relinked them as a connected service.

**Comment by voroninp** [comment]
@grasbdo , manually?

**Comment by grasbdo** [comment]
Yes.

**Comment by PawcioCNCProfi** [comment]
> You can use the legacy version of the upgrade-assistant in version 0.4.421302, see [here](https://learn.microsoft.com/en-us/dotnet/core/porting/upgrade-assistant-install-legacy) for more details.

Do you know if I can still download this anywhere? The link is broken.
